exonum-java-binding/launcher-plugins/README.md

```diff
@@ -53,19 +53,18 @@ runtimes:
 plugins:
   runtime: {}
   artifact: 
-    service_instance_name: "exonum_java_instance_plugin.JavaInstanceSpecLoader"
+    service_artifact_name: "exonum_java_instance_plugin.JavaInstanceSpecLoader"
```

Comment:
The key is the artifact identifier from `artifacts` section of the config.
```yaml
artifacts:
  service_artifact_name:
    runtime: java
    name: "com.exonum.examples:service-name:0.9.0-SNAPSHOT"
    spec:
      artifact_filename: "service-artifact-filename.jar"
```

Added a sample config and will probably try to rephrase readme cause it's not obvious for reader.

Reply:
Ok, so this format requires a second name for the artifact (in the example config, same as instance names). Is it worth it to add a second, shorter name instead of using the name an artifact already has?

Reply:
ping @popzxc 
The plugin does not change this part of the exonum-launcher. Though such implementation should be possible, as YAML supports strings as keys.

Reply:
Plugins section has the following format:
```yaml
plugins:
  runtime:
     $RUNTIME_NAME: $RUNTIME_PLUGIN_PYTHON_CLASS
  artifact:
    $SPEC_ARTIFACT_ALIAS: $SPEC_PLUGIN_PYTHON_CLASS
```

So, for example let's take this config:
```yaml
artifacts:
  cryptocurrency:
    runtime: sample_runtime
    name: "exonum-cryptocurrency:0.12.0"
  
instances:
  xnm-token:
    artifact: cryptocurrency
```

Here, we want to deploy artifact `cryptocurrency`. `cryptocurrency` here is a local artifact definition to be used within config (actual spec is stored inside, see `"exonum-cryptocurrency:0.12.0"`).

If we want to provide a plugin for a runtime, it will be:

```yaml
plugins:
  runtime:
     sample_runtime: our.package.SampleRuntimeSpecLoader
```

if we want also the spec for `cryptocurrency` artifact be loaded with plugin, then plugin section will become the following:

```yaml
plugins:
  runtime:
     sample_runtime: our.package.SampleRuntimeSpecLoader
  artifact:
    cryptocurrency: our.package.CryptocurrencySpecLoader
```

Why is it done that way? At least for `rust` runtime artifact names contain a version. So in order to just bump a version, you will have to update it everywhere (so it's up to N + 1 times -- N for every instance of that artifact in config, and 1 for plugin), with aliases it will be only one place.

And IMO it's just more readable.

Reply:
Thanks for clarifications, it makes sense!